lib/layout.js

```diff
@@ -215,8 +215,10 @@ El.calcSize = (el) => (
 
 El.calcScrollSize = (el) => (
   el === window
-    ? { w: el.scrollX, h: el.scrollY }
-    : { w: el.scrollLeft, h: el.scrollTop }
+   ? {
+    w: el.scrollX || el.pageXOffset || document.documentElement.scrollLeft,
+    h: el.scrollY || el.pageYOffset || document.documentElement.scrollTop
```

Comment:
I misread my own function pardon me; right we are resolving the check for `window` in this case not `el`.

This component uses `flexbox` and inline `svg` so unless you can find a browser that supports those but not  `pageX/YOffset` I am asking for that third fallback to be removed. Of course I'm happy to accept the `pageX/YOffset` fallback though. Make sense?